lib/linter-rust.coffee

```diff
@@ -108,6 +111,17 @@ class LinterRust
       directory = path.resolve path.join(directory, '..')
     return false
 
+  makeOpts: (PATH) ->
+    options = process.env;
 
+    switch process.platform
+      when "darwin" || "linux"
```

Comment:
Why not just use [path.delimiter](https://nodejs.org/api/path.html#path_path_delimiter)?